src/components/GoodsItem/GoodsItem.tsx

```diff
@@ -0,0 +1,9 @@
+type Props = {
+  good: string,
+};
+
+export const GoodsItem: React.FC<Props> = ({ good }) => (
+  <li className="Goods__item">
+    {good}
+  </li>
+);
```

Comment:
That bad idea to take `li` away from `ul`, plus you've forgot to explicitly indicate item key. I'm 100% sure Vova's gonna give a reject for this, so I'm gonna request changes before it happens)

Reply:
the key is delivered by props here, from the GoodsList component